src/validations/actionSchema.test.ts

```diff
@@ -0,0 +1,77 @@
+import ValidationError from 'yup/lib/ValidationError';
+
+import { extendLifecycleIdSchema } from './actionSchema';
+
+describe('extend lifecycle ID schema', () => {
+  it('should correctly validate a schema that is valid', () => {
+    expect(() =>
+      extendLifecycleIdSchema.validateSync({
+        newExpirationMonth: '2',
+        newExpirationDay: '15',
+        newExpirationYear: '2023',
+        newScope: 'A new scope',
+        newNextSteps: 'Some new next steps'
+      })
+    );
```

Comment:
This should test that it doesn't throw an error; right now, the test passes regardless of what `extendLifecycleIdSchema.validateSync()` does. To do this, the test should look like:
```ts
expect(() =>
      extendLifecycleIdSchema.validateSync({
        newExpirationMonth: '2',
        newExpirationDay: '15',
        newExpirationYear: '2023',
        newScope: 'A new scope',
        newNextSteps: 'Some new next steps'
      })
    ).not.toThrowError();
```

Reply:
I think the issue was that if an error is thrown from a callback that's passed to `expect()`, it doesn't necessarily fail the test without the `.not.toThrowError()` matcher. If an error is thrown at the top level of a test, i.e.
```
it("Fails from a thrown error", () => {
  throw new Error();
});
```
that'll be marked as a failing test.